slotfills/sections/fieldRouter.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
+import Field from '@/interfaces/field';
 import TextField from './text-field';
 import TextareaField from './textarea-field';
 import RichtextField from './richtext-field';
 
-const FieldRouter = ({
+type Props = {
```

Comment:
Prefer `interface` over `type`: https://blog.logrocket.com/types-vs-interfaces-in-typescript/

I've generally standardized on the naming convention of `Thing` for the component name and `ThingProps` for the props: https://github.com/alleyinteractive/wp-starter-plugin/blob/feature/typescript/components/image-picker/index.tsx#L5

Anecdotally, this seems to be a common pattern in the code I've seen.